app/controllers/users/approvals_controller.rb

```diff
@@ -22,4 +26,15 @@ def reject
     render "users/approvals/approve"
   end
 
+  def check_user
```

Comment:
@dansteele Not quite sure what you mean, I could add in a check to see if the user_id in params matches a user in the database, this would make the error message i could return slightly more specific. i.e. This user does not exist vs. You are not signed in as :user_id. I could also add to the error message who they are currently signed in as. 
But I don't think this is what you're asking haha.